<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonPause">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStop">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
    <widget class="QLabel" name="lightIndicatorBlue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lightIndicatorGreen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lightIndicatorRed">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="batteryChargeIndicator">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="mainDisplay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <widget class="QPushButton" name="newSessionButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>New Session</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sessionLogButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Session Log</string>
        </property>
       </widget>
       <widget class="QPushButton" name="dateAndTimeButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Date and Time</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="sessionTimerLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="sessionProgressBar">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QPushButton" name="CancelButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SubmitButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="CancelButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ViewSessionButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>ViewSession</string>
       </property>
      </widget>
      <widget class="QListWidget" name="sessionLogList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5"/>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="wavePlot" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="electrodeSelection">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="pcUiWidget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="detailsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
